src/main/java/ai/zuva/example/Example.java

```diff
@@ -0,0 +1,106 @@
+package ai.zuva.example;
+
+import ai.zuva.classification.ClassificationRequest;
+import ai.zuva.extraction.ExtractionRequest;
+import ai.zuva.fields.Field;
+import ai.zuva.files.ZdaiFile;
+import ai.zuva.api.ZdaiApiClient;
+import ai.zuva.language.LanguageRequest;
+import ai.zuva.ocr.OcrRequest;
+import ai.zuva.extraction.ExtractionData;
+import ai.zuva.extraction.ExtractionResults;
+import ai.zuva.fields.FieldMetadata;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
+
+public class Example {
+    public static void main(String[] args) throws Exception {
+        String token = System.getenv("ZUVA_TOKEN");
+        if (token == null) {
+            throw(new Exception("Environment variable ZUVA_TOKEN not set"));
+        }
+
+        String url = System.getenv("ZUVA_URL");
+        if (url == null) {
+            throw(new Exception("Environment variable ZUVA_URL not set"));
+        }
+
+        String documentPath = System.getenv("ZUVA_DEMO_DOC");
+        if (documentPath == null) {
+            // default to looking for the demo file in the current working directory
+            documentPath = "CANADAGOOS-F1Securiti-2152017.PDF2";
+        }
+
+        ZdaiApiClient client = new ZdaiApiClient(url, token);
+
+        ZdaiFile zdaiFile = ZdaiFile.submitFile(client, new File(documentPath));
+        System.out.println(String.format("Uploaded file with id %s expires at %s", zdaiFile.fileId, zdaiFile.expiration));
+
+        System.out.println("\nClassifying Document type:");
+        ClassificationRequest classificationRequest = ClassificationRequest.createClassificationRequest(client, zdaiFile);
+        System.out.println("Request ID: " + classificationRequest.requestId);
+
+        String status = StatusChecker.waitForStatus(() -> classificationRequest.getClassificationResult().status, 1, 60);
+        if (status.equals("complete")) {
+            System.out.println("Document type is: " + classificationRequest.getClassificationResult().classification);
```

Comment:
We probably want to rename this function to just `classificationRequest.getResult()` so it's less verbose and is more consistent with the `getResult` and `getResults` functions for language and extraction.